src/pages/saksoversikt/delt/Infolinje/arrayUtils.ts

```diff
@@ -0,0 +1,25 @@
+import { Månedsberegning, SimulertPeriode } from '~api/behandlingApi';
```

Comment:
Jeg syns både filstien og filnavnet her var litt rart.
Jeg klarer ikke å se helt knytningen mot `Infolinje`, og de to funksjonene som blir eksportert er ikke generelle utils for arrays 🤔

Hva med å bare legge det i f.eks. `src/pages/saksoversikt/utils`?

Reply:
Yes!